pkg/agent/endpoints/workload/handler.go

```diff
@@ -46,7 +45,12 @@ func (h *Handler) FetchX509SVID(_ *workload.X509SVIDRequest, stream workload.Spi
 	h.T.IncrCounterWithLabels([]string{"workload_api", "connections"}, 1, tLabels)
 	defer h.T.IncrCounterWithLabels([]string{"workload_api", "connections"}, -1, tLabels)
 
-	selectors := h.attest(pid)
+	config := attestor.Config{
+		Catalog: h.Catalog,
+		L:       h.L,
+		T:       h.T}
```

Comment:
Nit: would be nice to have the closing bracket on a new line

Reply:
Thanks! Will fix it.